demosplan/DemosPlanCoreBundle/Entity/User/Customer.php

```diff
@@ -122,6 +127,7 @@ class Customer extends CoreEntity implements UuidEntityInterface, CustomerInterf
      *
      * @ORM\Column(type="text", length=65535, nullable=false, options={"default":""})
      */
+    #[Assert\Length(min: 0, max: 4096, groups: [self::GROUP_UPDATE])]
```

Comment:
Why does this have 4096 when the column has a length of 65535?

Reply:
For the initial reasoning I can only speculate. Maybe the attribution is shown in a place where no more characters must be shown. I simply migrated the assertion from `CustomerResourceInterface`, which is currently (still) used to save platform settings and is ideally deleted during the remaining sprint time.

Reply:
Indeed, very interesting. Could you please change the length of the doctrine definition as well? Otherwise this is very confusing and with the assertions the values could not be longer anyways. `4096` seems legit for the usecase.

Reply:
The Column is of type 'text' and its length goes from >255 characters to <65536 characters - so the ORM Column does not change at all.

Reply:
It will probably be a MEDIUMTEXT with a length of  4096

Reply:
changed to match the others in 7ad88ce1c656f7e31dec06051bdda17a6769f001
but the technical limitation will still remain at 65535 as it is the TEXT limitation.

https://www.mysqltutorial.org/mysql-text/
TINYTEXT<TEXT<MEDIUMTEXT....

Reply:
Indeed, I made a mistake, MEDIUMTEXT is greater than TEXT. But thank you for the change